netket/experimental/operator/fermion.py

```diff
@@ -55,7 +59,11 @@ def create(
     Args:
         hilbert: The hilbert space
         site: the site on which this operator acts
-        sz: spin projection quantum number (e.g. sz=-0.5 for a spin-1/2 down)
+        sz: spin projection quantum number. This is the eigenvalue of
+            the corresponding spin-Z operator (e.g. `sz = ±1` for
```

Comment:
```suggestion
            the corresponding spin-Z Pauli operator (e.g. `sz = ±1` for
```

Reply:
why not adding Pauli?

Reply:
ah I think I accidentally eliminated the commit from the git history. I fixed it now